tomviz/python/tomviz/utils.py

```diff
@@ -31,50 +32,31 @@ def vtk_itk_map():
     if _vtk_to_itk_types is None:
         import itk
         _vtk_to_itk_types = {}
-        try:
-            _vtk_to_itk_types['vtkUnsignedCharArray'] = itk.Image.UC3
-        except:
-            pass
-
-        try:
-            _vtk_to_itk_types['vtkCharArray'] = itk.Image.SC3
-        except:
-            pass
-
-        try:
-            _vtk_to_itk_types['vtkUnsignedShortArray'] = itk.Image.US3
-        except:
-            pass
-
-        try:
-            _vtk_to_itk_types['vtkShortArray'] = itk.Image.SS3
-        except:
-            pass
-
-        try:
-            _vtk_to_itk_types['vtkUnsignedIntArray'] = itk.Image.UI3
-        except:
-            pass
-
-        try:
-            _vtk_to_itk_types['vtkIntArray'] = itk.Image.SI3
-        except:
-            pass
-
-        try:
-            _vtk_to_itk_types['vtkFloatArray'] = itk.Image.F3
-        except:
-            pass
-
-        try:
-            _vtk_to_itk_types['vtkDoubleArray'] = itk.Image.D3
-        except:
-            pass
+
```

Comment:
Sorry, missed your initial request for reviewing this.

I don't know if this code is less complex, but it is shorter. :-) LGTM